client/signup/config/flows-pure.js

```diff
@@ -14,7 +14,7 @@ export function generateFlows( {
 	getDestinationFromIntent = noop,
 	getDIFMSignupDestination = noop,
 	getDIFMSiteContentCollectionDestination = noop,
-	getStepperFlowDestination = noop,
+	// getStepperFlowDestination = noop,
```

Comment:
You can delete this along with the definition in `client/signup/config/flows.js`. This was used before we changed up the order and only existed for these two steps which are now gone.

Reply:
Done! Thank you for this tip @arcangelini